holidata/holidays/et-EE.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+from dateutil.easter import EASTER_WESTERN
+
+from .holidays import Locale
+
+
+class et_EE(Locale):
+    """
+    01-01: [NF] Uusaasta
+    02-24: [NF] Iseseisvuspäev, Eesti Vabariigi aastapäev
+    05-01: [NF] Kevadpüha
+    06-23: [NF] Võidupüha
+    06-24: [NF] Jaanipäev
+    08-20: [NF] Taasiseseisvumispäev
+    12-24: [NF] Jõululaupäev
+    12-25: [NF] Esimene jõulupüha
+    12-26: [NF] Teine jõulupüha
+    1 day before Easter: [NRV] Suur reede
+    Easter: [NRV] Ülestõusmispühade
```

Comment:
The correct name of the holiday is _Ülestõusmispühade 1. püha_.

Reply:
Ok